compiler/noirc_evaluator/src/ssa.rs

```diff
@@ -174,8 +174,6 @@ fn convert_generated_acir_into_circuit(
         ..
     } = generated_acir;
 
-    let locations = locations.clone();
```

Comment:
I found this while going through the non-inlined ACIR changes again as well. Decided to just clean it up in this PR.